BusinessCore.Tests/JVBBTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using CarManagement.Builders;
+using CarManagement.Models;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace BusinessCore.Tests
+{
+    [TestClass]
+    [TestCategory("JVBB Tests")]
+    public class JVBBTests
+    {
+        [TestMethod]
+        public void BrainDamagedBuilder_01()
```

Comment:
test method name must be more descriptive, please :)

Reply:
How about WrongBuildRequests for the method name?